src/engraving/dom/chord.cpp

```diff
@@ -1420,7 +1420,7 @@ double Chord::calcDefaultStemLength()
         // when the chord's magnitude is < 1, the stem length with mag can find itself below the middle line.
         // in those cases, we have to add the extra amount to it to bring it to a minimum.
         double upValue = m_up ? -1. : 1.;
-        double stemStart = startNote->pos().y();
+        double stemStart = startNote->pos().y() - startNote->offset().y();
```

Comment:
Same as https://github.com/musescore/MuseScore/pull/19559/files#r1341666459; `startNote->layoutData()->pos()` would be a more direct way of getting the desired value